monitors/command_test.go

```diff
@@ -200,3 +200,36 @@ func TestDiffMonitorsWithScopes(t *testing.T) {
 		t.Errorf("expected:\n%s\n, output:\n%s\n", expected, diff)
 	}
 }
+
+func TestMonitorLoadRules(t *testing.T) {
+	tmpFile, err := os.CreateTemp("", "")
+	if err != nil {
+		t.Errorf("should not raise error: %v", err)
+	}
+	defer os.Remove(tmpFile.Name())
```

Comment:
testing.T.TempDir is useful.

Reply:
I had tried it on https://github.com/mackerelio/mkr/pull/622/commits/8df6538fbd93beddeec7de718d1282d5f200910d , https://github.com/mackerelio/mkr/pull/622/commits/3295eed0f77b3460d2fe25143831d47360b02eae , and https://github.com/mackerelio/mkr/pull/622/commits/db949f894855a61decdeccff129df617a33e9012 but all of them went failed on Windows (I saw https://github.com/golang/go/issues/50510 )
It would be better to use it but I keep CreateTemp code at this time for Windows.